target/linux/ramips/dts/mt7621_zte_mf286c3.dts

```diff
@@ -0,0 +1,314 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+    compatible = "zte,mf286c3", "mediatek,mt7621-soc";
+    model = "ZTE MF286C3";
+
+    chosen {
+        bootargs = "console=ttyS0,57600n8";
+        bootargs-override = "console=ttyS0,57600n8";
+    };
+
+    leds {
+        compatible = "gpio-leds";
+        pinctrl-names = "default";
+        pinctrl-0 = <&led_pins>;
+
+        led_p0_0: p0_0 { // sig00
+            label = "sig_0";
+            gpios = <&aw9523 0 GPIO_ACTIVE_LOW>;
+        };
+
+        led_p0_1: p0_1 { // sig01
+            label = "sig_1";
+            gpios = <&aw9523 1 GPIO_ACTIVE_LOW>;
+        };
+
+        led_p0_2: p0_2 { // sig02
+            label = "sig_2";
+            gpios = <&aw9523 2 GPIO_ACTIVE_LOW>;
+        };
+
+        led_p0_3: p0_3 { // sig03
+            label = "sig_3";
+            gpios = <&aw9523 3 GPIO_ACTIVE_LOW>;
+        };
+
+        led_p0_5: p0_5 { // rgb05-red
+            label = "rgb-red";
+            gpios = <&aw9523 5 GPIO_ACTIVE_LOW>;
+        };
+
+        led_p0_6: p0_6 { // power_06
+            label = "power";
+            gpios = <&aw9523 6 GPIO_ACTIVE_LOW>;
+            default-state = "on";
+        };
+
+        led_p1_0: p1_0 { // rgb10-blue
+            label = "rgb-blue";
+            gpios = <&aw9523 8 GPIO_ACTIVE_LOW>;
+        };
+
+        led_p1_1: p1_1 { // sig11
+            label = "sig_5";
+            gpios = <&aw9523 9 GPIO_ACTIVE_LOW>;
+        };
+
+        led_p1_2: p1_2 { // wifi12
+            label = "wifi";
+            gpios = <&aw9523 10 GPIO_ACTIVE_LOW>;
+        };
+
+        led_p1_3: p1_3 { // rgb13-green
+            label = "rgb-green";
+            gpios = <&aw9523 11 GPIO_ACTIVE_LOW>;
+        };
+
+        led_p1_4: p1_4 { // phone14
+            label = "phone";
+            gpios = <&aw9523 12 GPIO_ACTIVE_LOW>;
```

Comment:
New style would be better instead of the label.
```
			color = <LED_COLOR_ID_WHITE>;
			function = LED_FUNCTION_STATUS;
```
https://github.com/torvalds/linux/blob/master/include/dt-bindings/leds/common.h

And please sort the labels and keys by the gpios.